src/stylesheets/layouts/_question.scss

```diff
@@ -0,0 +1,80 @@
+.question {
+  background-color: $color-mint;
+  display: flex;
+  justify-content: center;
+  margin-top: 1.5rem;
+
+  &__wrapper {
+    position: relative;
+    @include flex;
+    width: 100%;
+    height: 100%;
+    max-width: 77rem;
+    margin: 0rem;
+    padding: 2.5rem 1.5rem;
```

Comment:
🔤

Reply:
Also - same note - watch how much you're using flexbox

Reply:
resolved in pr#4